src/sentry/static/sentry/app/views/releases/pullRequestLink.jsx

```diff
@@ -10,34 +10,54 @@ class PullRequestLink extends React.Component {
     inline: PropTypes.bool,
   };
 
-  getUrl = () => {
-    if (this.props.repository.provider.id === 'github') {
+  get providerId() {
+    if (!this.props.repository.provider) {
+      return null;
+    }
+
+    let id = this.props.repository.provider.id;
+    if (id.indexOf(':') > -1) {
+      return id.split(':').pop();
+    }
+    return id;
+  }
+
+  get url() {
```

Comment:
So with this... why is this not done on the backend rather than the frontend? Normally don't we want to do most of the data manipulation like forming urls on the backend and then just display it on the frontend.

Reply:
I agree that building the URLs clientside isn't ideal. I would like to move it to the serverside, but wanted to get this problem fixed before changing backend API responses.

Reply:
I created APP-730 to capture the desired shift in logic.